[IMP] stock_picking_package_info: New button in transfer wizard.

alfredoavanzosc opened the pull request:
[IMP] stock_picking_wave_package_info: take into account new button.

Modificado el módulo "stock_picking_package_info" a petición de Ana, para meter un nuevo botón en el wizard de trasnferencia de albaranes. También se ha tenido que modificar el módulo "stock_picking_wave_package_info", por dependencia del módulo anterior.

Commit:
[IMP] stock_picking_package_info: New button in transfer wizard.
[IMP] stock_picking_wave_package_info: take into account new button.

Commit:
[IMP] stock_picking_package_info: New button in transfer wizard.
[IMP] stock_picking_wave_package_info: take into account new button.

alfredoavanzosc commented:
Pedro... revisa si quieres los 2 módulos modificados, pero no hagas el MERGE, ya que Ana me ha pedido más modificaciones para estos 2 módulos. Muchas gracias.

pedrobaeza commented on stock_picking_package_info/README.rst:
```diff
@@ -14,6 +13,9 @@ logical unit. The values of these fields can be modified.
 In the field "real weight", shown the value of total estimated weight field.
 The value of this field can be changed.
 
+In the transfer wizard pickings, has gotten the new "Record" button, allowing
```
You have a new button "Record", that allows...

pedrobaeza commented on stock_picking_package_info/README.rst:
```diff
@@ -14,6 +13,9 @@ logical unit. The values of these fields can be modified.
 The value of this field can be changed.
 
+In the transfer wizard pickings, has gotten the new "Record" button, allowing
+only modify operations of the picking
```
No termino de entender qué hace este botón. ¿Puedes explicarlo mejor?

Commit:
[IMP] stock_picking_package_info: Remove compute in "packages" field of stock.picking
[IMP] stock_picking_wave_package_info: Remove compute in "packages" field of stock.picking

Commit:
[IMP] stock_picking_package_info: Remove compute in "packages" field of stock.picking
[IMP] stock_picking_wave_package_info: Remove compute in "packages" field of stock.picking

alfredoavanzosc commented:
Se ha quitado el compute del campo "packages" para los objetos "stock.picking", y "stock.picking.wave"

pedrobaeza commented on stock_picking_package_info/README.rst:
```diff
@@ -14,6 +13,9 @@ logical unit. The values of these fields can be modified.
 The value of this field can be changed.
 
+In the transfer wizard pickings, you have a new button "Record", that allows
+only modify datas of operations of the picking
```
Sigo sin entender qué hace el botón "Record".

anajuaristi commented:
Es una cosa muy simple.
El standar de Odoo, cuando pulsas el wizard transferir, en realidad lo que hace es cargar la tabla operations (aunque está oculta a la vista en el albarán)
Las 2 opciones que te da son transferir y cancelar. Con transferir, haces la transferencia. Con cancelar, pierdes lo que hubieses hecho en el wizard. 
Suponte que tienes 10 líneas y asignas paquetes a 5, por ejemplo. Pero... te das cuenta que te falta algo o quieres volver al albarán antes de transferir. Con el standar pierdes todo si pulsas cancelar y el albarán queda realizado si pulsas transferir. 
Con el nuevo botón "record" te graba en la tabla de operaciones lo que has hecho en el wizard, de tal forma que la siguiente vez que lo lanzas, lo que hiciste anteriormente ya está ahí.
Permite ir montando paquetes poco a poco, sobre todo en albaranes que son muy largos o waves con muchas líneas... 
No impacta para nada en el standar porque los 2 botones cancel y transferir siguen funcionando exactamente igual. Lo único que dota al wizard de trasnferencia la posibilidad de ir asignando cosas por partes si lo necesitas.

pedrobaeza commented:
Ahora si lo entiendo, gracias.

La etiqueta "Record" para el botón no sería entonces lo más indicado, si no "Save for later" o algo así.

Pues es curioso, porque justo lo que describes es lo que hace el interfaz de código de barras estándar: dejar guardada esa información. ¿Seguro que no se quedan guardados los cambios que hagas? ¿No son campos related?

Por otro lado, ¿es éste el módulo adecuado en caso de que no lo haga?

anajuaristi commented:
No. No se quedan.
Si haces cualquier cosa en el wizard de transfer y pulsas cancelar, cuando lo abres ya no hay nada... tienes que volver a empezar de cero.
El wizard es un objeto diferente a la tabla operaciones y no creo que son campos related. No te sabría decir. 

Yo veo que es el módulo adecuado básicamente porque lo que más puedes hacer por partes son paquetes pero... eso ya, no sé. Si lo ves más adecuado en otro sitio.... La verdad es que debería venir de base ya esta opción. Da muchísima versatilidad. Te permite jugar mucho y cambiar todo antes de decidir lanzarte a confirmar la transferencia

anajuaristi commented:
@pedrobaeza mergeamos entonces?

pedrobaeza commented:
No, cuando haga el cambio de nombre que he pedido.

Commit:
Merge branch 'odoomrp-stock_picking_package_info' of https://github.com/alfredoavanzosc/odoomrp-wip-1 into odoomrp-stock_picking_package_info

Commit:
[FIX] stock_picking_package_info: Rename the "Record" to "Save to later" button and rename its function.
[FIX] stock_picking_wave_package_info: Modified by renaming function in the previous module.

alfredoavanzosc commented:
Cambiado el nombre del botón, también renombrada su función, y cambiada la traducción. Subidos los cambios.

pedrobaeza commented:
:+1: cuando Travis dé verde.
